src/login/redux/reducers/resetPasswordReducer.js

```diff
@@ -11,6 +12,21 @@ let resetPasswordReducer = (state = data, action) => {
         ...state,
         ...action.payload
       };
+    case postActions.SAVE_RESET_PASSWORD_VERIFY_EMAIL_CODE:
+      return {
+        ...state,
+        ...action.payload
+      };
+    case postActions.POST_RESET_PASSWORD_VERIFY_EMAIL:
+      return {
+        ...state,
+        ...action.payload
+      };
+    case postActions.POST_RESET_PASSWORD_VERIFY_EMAIL_SUCCESS:
```

Comment:
It looks like the email code is included in the response here and gets listed twice in the store (code and email_code). You could exchange the state.resetPassword.code with action.payload.email_code to avoid having the code stored twice.

Reply:
yes! I forgot to remove it.